frontend/routes/manage/Questionnaire.vue

```diff
@@ -0,0 +1,116 @@
+    </div>
+</template>
+
+<script>
+    import Button    from "../../components/base/Button.vue";
+    import Dropdown  from "../../components/base/Dropdown.vue";
+    import TextArea  from "../../components/base/TextArea.vue";
+    import TextInput from "../../components/base/TextInput.vue";
+    import Table     from "../../components/base/Table.vue";
+
+    export default {
+        components: {
+            Button,
+            Dropdown,
+            TextArea,
+            TextInput,
+            Table
+        },
+        data() {
+            return {
+                name: "Questionnaire",
+                tableRows: [
+                    {
+                        id: 1,
+                        firstName: "Claire",
+                        lastName: "Gallup",
+                        email: "claire.j.gallup@gmail.com",
+                        status: "Confirmed",
+                        checkedIn: "Yes",
+                        appliedOn: "7/5/21",
+                    },
+                    {
+                        id: 2,
+                        firstName: "Claire",
+                        lastName: "Gallup",
+                        email: "claire.j.gallup@gmail.com",
+                        status: "Confirmed",
+                        checkedIn: "Yes",
+                        appliedOn: "7/5/21",
+                    },
+                    {
+                        id: 3,
+                        firstName: "Claire",
+                        lastName: "Gallup",
+                        email: "claire.j.gallup@gmail.com",
+                        status: "Confirmed",
+                        checkedIn: "Yes",
+                        appliedOn: "7/5/21",
+                    },
+                    {
+                        id: 4,
+                        firstName: "Claire",
+                        lastName: "Gallup",
+                        email: "claire.j.gallup@gmail.com",
+                        status: "Confirmed",
+                        checkedIn: "Yes",
+                        appliedOn: "7/5/21",
+                    },
+                    {
+                        id: 5,
+                        firstName: "Claire",
+                        lastName: "Gallup",
+                        email: "claire.j.gallup@gmail.com",
+                        status: "Confirmed",
+                        checkedIn: "Yes",
+                        appliedOn: "7/5/21",
+                    }
+                ],
+                tableColumns: {
+                    firstName: "First Name",
+                    lastName: "Last Name",
+                    email: "Email",
+                    status: "Status",
+                    checkedIn: "Checked In?",
+                    appliedOn: "Applied On",
+                }
+            }
+        },
+    };
+</script>
+
+<style lang="scss" scoped>
+.space-between {
+    display: flex;
+    justify-content: space-between;
+}
+</style>
```

Comment:
nit: missing newline at end of file
```suggestion
</style>

```